scripts/deployPriceFeed.ts

```diff
@@ -1,4 +1,12 @@
+import { LedgerSigner } from '@anders-t/ethers-ledger'
+const IS_LEDGER_SIGNER = true
+const LEDGER_ACCOUNT_PATH = "44'/60'/0'/0/0"
+
 export async function deploy(network_name, fxCurrency, hre) {
+  const ERC20 = await hre.ethers.getContractFactory('MockToken')
+  const provider = ERC20.signer.provider
+  const signer = IS_LEDGER_SIGNER ? await new LedgerSigner(provider, LEDGER_ACCOUNT_PATH) : ERC20.signer
+
   const networks = {
```

Comment:
It looks like we still need to add `EURS` & `DAI` here?

Reply:
> It looks like we still need to add `EURS` & `DAI` here?

@schystz we could, though we're not adding EURS in lending market at the moment. DAI is already added in our LM and it has DAI/ETH chainlink oracle.

Reply:
Oh wasn't this for the token pairs? I was thinking for `EURS:USDC` and `DAI:USDC` eth pricefeed oracle

Reply:
`DeployHLP.ts` is the one for the token pair, and yes we might need to add those as well. I'll add them too in `DeployHLP.ts` @schystz Thanks for checking